urllib3/util/connection.py

```diff
@@ -99,3 +106,39 @@ def _set_socket_options(sock, options):
 
     for opt in options:
         sock.setsockopt(*opt)
+
+
+def family_filter():
```

Comment:
Not sure "filter" is the right word here. Maybe `get_address_family()`?

Filter makes me think that it takes some input and returns a subset output.